oci/spec_opts_unix.go

```diff
@@ -478,25 +491,45 @@ func getAllCapabilities() []string {
 
 var errNoUsersFound = errors.New("no users found")
 
-func getUIDGIDFromPath(root string, filter func(user.User) bool) (uid, gid uint32, err error) {
```

Comment:
Don’t combine this into a single function. It’s a separate file with different failure scenarios.

Reply:
Please update and rebase when you have time

Reply:
@crosbymichael Done! Ptal